opennlp-tools/src/test/java/opennlp/tools/cmdline/CLITest.java

```diff
@@ -27,6 +27,7 @@
 public class CLITest {
 
   private static class ExitException extends SecurityException {
+    private static final long serialVersionUID = -1569185042676202805L;
```

Comment:
Lets not add serialVersionUID, the default is fine, and if we need to change it we can do it.